Paxifax.cs

```diff
@@ -21,13 +21,186 @@
 using System.Reflection;
 using System.Linq;
 
+// FIXME use javadoc-style comments to describe the API
 namespace Pax {
 
-  // FIXME use javadoc-style comments to describe the API
-  public interface PacketProcessor {
+  // A more abstract interface to packet processors.
+  public interface Abstract_PacketProcessor {
```

Comment:
I don't know what you think about this, but a common pattern is to name interfaces e.g. `IPacketProcessor`.